lib/al/Library/Player/PlayerHolder.h

```diff
@@ -30,4 +39,48 @@ class PlayerHolder {
     al::PadRumbleKeeper* getPadRumbleKeeper(s32 index) const;
 };
 
+s32 getPlayerNumMax(const al::LiveActor*);
+s32 getPlayerNumMax(const al::PlayerHolder*);
+s32 getAlivePlayerNum(const al::LiveActor*);
+s32 getAlivePlayerNum(const al::PlayerHolder*);
+al::LiveActor* getPlayerActor(const al::LiveActor*, s32);
+al::LiveActor* getPlayerActor(const al::PlayerHolder*, s32);
+const sead::Vector3f& getPlayerPos(const al::LiveActor*, s32);
+const sead::Vector3f& getPlayerPos(const al::PlayerHolder*, s32);
+al::LiveActor* tryGetPlayerActor(const al::LiveActor*, s32);
+al::LiveActor* tryGetPlayerActor(const al::PlayerHolder*, s32);
+bool isPlayerDead(const al::LiveActor*, s32);
+bool isPlayerDead(const al::PlayerHolder*, s32);
+bool isPlayerAreaTarget(const al::LiveActor*, s32);
+bool isPlayerAreaTarget(const al::PlayerHolder*, s32);
+al::LiveActor* tryFindAlivePlayerActorFirst(const al::LiveActor*);
+al::LiveActor* tryFindAlivePlayerActorFirst(const al::PlayerHolder*);
+al::LiveActor* findAlivePlayerActorFirst(const al::LiveActor*);
+al::LiveActor* findAlivePlayerActorFirst(const al::PlayerHolder*);
+al::PadRumbleKeeper* getPlayerPadRumbleKeeper(const al::LiveActor*, s32);
+const al::PadRumbleKeeper& getPlayerPort(const al::PlayerHolder*, s32);
+const al::PadRumbleKeeper& getPlayerPort(const al::LiveActor*, s32);
+al::LiveActor* findAlivePlayerActorFromPort(const al::PlayerHolder*, s32);
+al::LiveActor* tryFindAlivePlayerActorFromPort(const al::PlayerHolder*, s32);
+al::LiveActor* findAlivePlayerActorFromPort(const al::LiveActor*, s32);
+al::LiveActor* tryFindAlivePlayerActorFromPort(const al::LiveActor*, s32);
+s32 findNearestPlayerId(const al::LiveActor*, f32);
+al::LiveActor* findNearestPlayerActor(const al::LiveActor*);
+al::LiveActor* tryFindNearestPlayerActor(const al::LiveActor*);
+const sead::Vector3f& findNearestPlayerPos(const al::LiveActor*);
+const sead::Vector3f& tryFindNearestPlayerPos(sead::Vector3f*, const al::LiveActor*);
+const sead::Vector3f& tryFindNearestPlayerDisatanceFromTarget(f32*, const al::LiveActor*, const sead::Vector3f&);
+bool isNearPlayer(const al::LiveActor*, f32);
+bool isNearPlayerH(const al::LiveActor*, f32);
+bool isNearPlayerHCondition(const al::LiveActor*, f32, bool (*)(const al::LiveActor*));
+const sead::Vector3f& getFarPlayerPosMaxX(const al::LiveActor*);
+const sead::Vector3f& getFarPlayerPosMinX(const al::LiveActor*);
```

Comment:
Seems weird, but I can't find anything that would point this out as wrong...

Reply:
That definitely wouldn't be something that you should be able to modify. Weird functions regardless.